orchestration/src/test/java/com/sap/ai/sdk/orchestration/client/OrchestrationUnitTest.java

```diff
@@ -78,15 +81,40 @@ public class OrchestrationUnitTest {
 
   @BeforeEach
   void setup(WireMockRuntimeInfo server) {
+
+    stubFor(
+        get(urlPathEqualTo("/v2/lm/deployments"))
+            .withHeader("AI-Resource-Group", equalTo("default"))
+            .willReturn(
+                aResponse()
```

Comment:
```suggestion
                ok()
```

Not sure if it makes any difference though